completion/available/wpscan.completion.bash

```diff
@@ -1,15 +1,62 @@
 #!/usr/bin/bash
 
 if command -v wpscan > /dev/null; then
```

Comment:
can you convert this to `_command exists`? if its a binary, you can also use `_binary_exists`

Reply:
Done